### PR TITLE
updates json rpcs

### PR DESCRIPTION
**Pull request by dawnkelly09**
### Description

- Adds eth_maxPriorityFeePerGas and eth_newPendingTransactionFilter to the supported Ethereum RPC list. 
- Adds section for known unsupported Ethereum RPC methods
- Minor punctuation changes per guidelines

### Checklist

- [ x] I have added a label to this PR 🏷️
- [ x] I have run my changes through Grammarly

### CODE REVIEW

File: builders/ethereum/json-rpc/eth-rpc.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The basic JSON-RPC methods from the Ethereum API supported by Moonbeam are:
 - **[eth_protocolVersion](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_protocolversion){target=\_blank}** — returns `1` by default
 - **[eth_syncing](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_syncing){target=\_blank}** — returns an object with data about the sync status or `false`
 - **[eth_hashrate](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_hashrate){target=\_blank}** — returns `"0x0"` by default
-- **[eth_coinbase](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_coinbase){target=\_blank}** — returns the latest block author. Not necessarily a finalized block
+- **[eth_coinbase](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_coinbase){target=\_blank}** — returns the latest block author. Not necessarily a finalized block.
```
Comment (eshaben) [inline builders/ethereum/json-rpc/eth-rpc.md:21]:
```suggestion
- **[eth_coinbase](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_coinbase){target=\_blank}** — returns the latest block author. Not necessarily a finalized block
```

File: builders/ethereum/json-rpc/eth-rpc.md [hunk 2]
```diff
@@ -49,10 +50,21 @@ The basic JSON-RPC methods from the Ethereum API supported by Moonbeam are:
 - **[eth_getLogs](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getlogs){target=\_blank}** — returns an array of all logs matching a given filter object
 - **[eth_newFilter](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_newfilter){target=\_blank}** — creates a filter object based on the input provided. Returns a filter ID
 - **[eth_newBlockFilter](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_newblockfilter){target=\_blank}** — creates a filter in the node to notify when a new block arrives. Returns a filter id
+- **[eth_newPendingTransactionFilter](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_newpendingtransactionfilter){target=\_blank}** - creates a filter in the node to notify when new pending transactions arrive. Returns a filter id
```
Comment (eshaben) [inline builders/ethereum/json-rpc/eth-rpc.md:53]:
```suggestion
- **[eth_newBlockFilter](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_newblockfilter){target=\_blank}** — creates a filter in the node to notify when a new block arrives. Returns a filter ID
- **[eth_newPendingTransactionFilter](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_newpendingtransactionfilter){target=\_blank}** - creates a filter in the node to notify when new pending transactions arrive. Returns a filter ID
```

File: builders/ethereum/json-rpc/eth-rpc.md [hunk 3]
```diff
@@ -49,10 +50,21 @@ The basic JSON-RPC methods from the Ethereum API supported by Moonbeam are:
...
 
+## Unsupported Ethereum JSON-RPC Methods {: #unsupported-rpc-methods }
+
+ Moonbeam does not support the following Ethereum API JSON-RPC methods:
+
+ - **[eth_getProof](https://docs.alchemy.com/reference/eth-getproof){target=\_blank}** - returns the account and storage values of the specified account including the Merkle-proof
+ - **[eth_blobBaseFee](https://www.quicknode.com/docs/ethereum/eth_blobBaseFee){target=\_blank}** - returns the expected base fee for blobs in the next block
+ - **[eth_createAccessList](https://docs.alchemy.com/reference/eth-createaccesslist){target=\_blank}** - creates an EIP2930 type `accessList` based on a given transaction object
+ - **[eth_sign](https://support.metamask.io/privacy-and-security/what-is-eth_sign-and-why-is-it-a-risk/){target=\_blank}** - presents a security risk due to signing an arbitrary hash which can be fraudulently applied to other transactions
```
Comment (eshaben) [inline builders/ethereum/json-rpc/eth-rpc.md:65]:
```suggestion
 - **[eth_sign](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_sign){target=\_blank}** - INSERT_DESCRIPTION. Presents a [security risk](https://support.metamask.io/privacy-and-security/what-is-eth_sign-and-why-is-it-a-risk/){target=\_blank} due to signing an arbitrary hash which can be fraudulently applied to other transactions
```

Can you please just add a brief description here before going into why it's not supported?

File: builders/ethereum/json-rpc/eth-rpc.md [hunk 4]
```diff
@@ -49,10 +50,21 @@ The basic JSON-RPC methods from the Ethereum API supported by Moonbeam are:
...
+ - **[eth_signTransaction](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_signtransaction){target=\_blank}** - presents a security risk due to signing a transaction to be sent at a later time
```
Comment (eshaben) [inline builders/ethereum/json-rpc/eth-rpc.md:66]: same here, please add a brief description first

File: builders/ethereum/json-rpc/eth-rpc.md [hunk 5]
```diff
@@ -49,10 +50,21 @@ The basic JSON-RPC methods from the Ethereum API supported by Moonbeam are:
...
 - **[eth_getFilterLogs](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getfilterlogs){target=\_blank}** — returns an array of all the logs matching the filter with a given ID
 - **[eth_uninstallFilter](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_uninstallfilter){target=\_blank}** — uninstall a filter with a given ID. It should be used when polling is no longer needed. Filters timeout when they are not requested using `eth_getFilterChanges` after some time
 
+## Unsupported Ethereum JSON-RPC Methods {: #unsupported-rpc-methods }
+
+ Moonbeam does not support the following Ethereum API JSON-RPC methods:
+
+ - **[eth_getProof](https://docs.alchemy.com/reference/eth-getproof){target=\_blank}** - returns the account and storage values of the specified account including the Merkle-proof
+ - **[eth_blobBaseFee](https://www.quicknode.com/docs/ethereum/eth_blobBaseFee){target=\_blank}** - returns the expected base fee for blobs in the next block
+ - **[eth_createAccessList](https://docs.alchemy.com/reference/eth-createaccesslist){target=\_blank}** - creates an EIP2930 type `accessList` based on a given transaction object
```
Comment (eshaben) [inline builders/ethereum/json-rpc/eth-rpc.md:64]:
```suggestion
 - **[eth_createAccessList](https://docs.alchemy.com/reference/eth-createaccesslist){target=\_blank}** - creates an EIP-2930 type `accessList` based on a given transaction object
```

File: builders/ethereum/json-rpc/eth-rpc.md [hunk 6]
```diff
@@ -39,6 +39,7 @@ The basic JSON-RPC methods from the Ethereum API supported by Moonbeam are:
 - **[eth_sendRawTransaction](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_sendrawtransaction){target=\_blank}** — creates a new message call transaction or a contract creation for signed transactions. Returns the transaction hash or the zero hash if the transaction is not yet available
 - **[eth_call](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_call){target=\_blank}** — executes a new message call immediately without creating a transaction on the blockchain, returning the value of the executed call
 - **[eth_estimateGas](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_estimategas){target=\_blank}** — returns an estimated amount of gas necessary for a given transaction to succeed. You can optionally specify a `gasPrice` or `maxFeePerGas` and `maxPriorityFeePerGas`
+- **[eth_maxPriorityFeePerGas](https://docs.alchemy.com/reference/eth-maxpriorityfeepergas){target=\_blank}** - returns an estimate of how much priority fee, in wei, is needed for inclusion in a block
```
Comment (eshaben) [inline builders/ethereum/json-rpc/eth-rpc.md:42]:
```suggestion
- **[eth_maxPriorityFeePerGas](https://docs.alchemy.com/reference/eth-maxpriorityfeepergas){target=\_blank}** - returns an estimate of how much priority fee, in Wei, is needed for inclusion in a block
```

**Review by eshaben — COMMENTED** [review]
Thank you for taking the time to check each of these out! 🙌 

Left some comments on minor things 🙂

File: builders/ethereum/json-rpc/eth-rpc.md (hunk 4 above)
Comment (dawnkelly09) [inline builders/ethereum/json-rpc/eth-rpc.md:66]:
```suggestion
 - **[eth_signTransaction](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_signtransaction){target=\_blank}** - allows the user to sign a transaction to be sent at a later time. It is rarely used due to associated security risks
```

**Review by dawnkelly09 — COMMENTED** [review]
added description

**Review by dawnkelly09 — COMMENTED** [review]
changes committed per review

File: builders/ethereum/json-rpc/eth-rpc.md [hunk 7]
```diff
@@ -48,11 +49,22 @@ The basic JSON-RPC methods from the Ethereum API supported by Moonbeam are:
...
 
+## Unsupported Ethereum JSON-RPC Methods {: #unsupported-rpc-methods }
+
+ Moonbeam does not support the following Ethereum API JSON-RPC methods:
+
+ - **[eth_getProof](https://docs.alchemy.com/reference/eth-getproof){target=\_blank}** - returns the account and storage values of the specified account including the Merkle-proof
+ - **[eth_blobBaseFee](https://www.quicknode.com/docs/ethereum/eth_blobBaseFee){target=\_blank}** - returns the expected base fee for blobs in the next block
+ - **[eth_createAccessList](https://docs.alchemy.com/reference/eth-createaccesslist){target=\_blank}** - creates an EIP-2930 type `accessList` based on a given transaction object
+ - **[eth_sign](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_sign){target=\_blank}** - allows user to sign an arbitrary hash to be sent at a later time. Presents a [security risk](https://support.metamask.io/privacy-and-security/what-is-eth_sign-and-why-is-it-a-risk/){target=\_blank} as the arbitrary hash can be fraudulently applied to other transactions
```
Comment (eshaben) [inline builders/ethereum/json-rpc/eth-rpc.md:65]:
```suggestion
 - **[eth_sign](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_sign){target=\_blank}** - allows the user to sign an arbitrary hash to be sent at a later time. Presents a [security risk](https://support.metamask.io/privacy-and-security/what-is-eth_sign-and-why-is-it-a-risk/){target=\_blank} as the arbitrary hash can be fraudulently applied to other transactions
```

**Review by eshaben — COMMENTED** [review]
one tiny thing, but other than that looks good!

**Review by eshaben — APPROVED** [review]
LGTM! ✨

**Pull request merged**